src/application.rs

```diff
@@ -235,18 +243,30 @@ pub trait ApplicationHandler {
     /// ### Web
     ///
     /// On Web, the [`suspended()`] method is called in response to a [`pagehide`] event if the
-    /// page is being restored from the [`bfcache`] (back/forward cache) - an in-memory cache that
+    /// page is being moved/stored in the [`bfcache`] (back/forward cache) - an in-memory cache that
```

Comment:
```suggestion
    /// page is being stored in the [`bfcache`] (back/forward cache) - an in-memory cache that
```
Should be enough I believe.